packages/aws-otel-collector/aws-otel-collector.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=AWS OTEL collector
```

nit: not sure of this, but the project page gives "ADOT Collector" as the short form of the name
```suggestion
Description=ADOT collector
```

---

Yep, that was the name given in the default service, but I changed it to `AWS OTEL collector` to match the package name we are adding to the core-kit. 

I prefer keeping it as `AWS OTEL collector` due to that reason, but let me know if you feel strongly about changing it to `ADOT collector`!